src/main/groovy/org/mockito/release/internal/gradle/NotableNotesFetcherTask.java

```diff
@@ -0,0 +1,34 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.TaskAction;
+import org.mockito.release.internal.gradle.util.DefaultReleaseNotesSerializer;
+import org.mockito.release.internal.gradle.util.ReleaseNotesSerializer;
+import org.mockito.release.notes.generator.ReleaseNotesGenerator;
+import org.mockito.release.notes.generator.ReleaseNotesGenerators;
+import org.mockito.release.notes.model.ReleaseNotesData;
+
+import java.util.Collection;
+
+public class NotableNotesFetcherTask extends DefaultTask {
+
+    //TODO documentation
+    private final NotesGeneration notesGeneration = new NotesGeneration();
+
+    public NotesGeneration getNotesGeneration() {
```

Comment:
You probably want to use @Nested property here (check out Gradle docs on this :).